raiden/tests/integration/transfer/test_refundtransfer.py

```diff
@@ -229,15 +229,15 @@ def test_refund_transfer(
         assert secrethash in state_from_raiden(app0.raiden).payment_mapping.secrethashes_to_task
 
         # make sure that app1 sent a lock expired message for the secrethash
-        send_lock_expired = raiden_events_must_contain_entry(
+        send_lock_expired = raiden_events_search_for_item(
             app1.raiden,
             SendLockExpired,
             {'secrethash': secrethash},
         )
         assert send_lock_expired
         # make sure that app0 never got it
         state_changes = app0.raiden.wal.storage.get_statechanges_by_identifier(0, 'latest')
```

Comment:
in a follow up PR I will fix this and use `raiden_state_changes_search_for_item`